### PR TITLE
Automate the PVC migration process.

### PR DESCRIPTION
**Pull request by WadeBarnes**

### CODE REVIEW

**Comment by WadeBarnes** [comment]
@jefkel, Please have a look and let me know if you see any necessary changes before this gets merged.

File: docker/migrate [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+usage () {
+  cat <<-EOF
+
+  ======================================================================================
+  This container and script allow you to easily migrate files from one
+  PVC to another.  By default when the container is deployed the source and 
+  destination PVCs are mounted to /source/ and /target/ respectively.
+
+  When run in container mode the script will sleep for ${SLEEP} to keep
+  the conatiner alive while you run the script in it's default sync mode
```
Comment (jefkel) [inline docker/migrate]:
```suggestion
  the container alive while you run the script in it's default sync mode
```

File: openshift/manage [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+      - Deploy the PVC migrator build configuration.
+
+    migrate <hostPodName> <pvcName> <pvcType> <pvcSize>
+      - Migrate files from one PVC to another of the same name.
+        Good for migrating from one storage class to another, or to a new PVC of a different size.
+
+      - Required parameters:
+        - hostPodName: The friendly name of the host pod.  Typically, this is the same as the deployment configuration name.
+        - pvcName: The name of the PVC to be migrated.
+        - pvcType: The PVC class to use for the new PVC.  One of gluster-file, gluster-block, gluster-file-db, or nfs-backup.
```
Comment (jefkel) [inline openshift/manage]:
```suggestion
        - pvcType: The PVC class to use for the new PVC.  One of gluster-file, gluster-block, or gluster-file-db.
```

**Comment by jefkel** [comment]
Quick check looks like you've got the nfs-backup StorageClass as an option for your auto migrate.  suggestion to remove made. (for nfs backup, we wont' be able to do the "keep the same name" piece.

I'll get some test migrations done with this tonight to see if anything else jumps out (not expecting any issues, just want to see if I can break it before I merge.)

**Comment by WadeBarnes** [comment]
@jefkel, All references to the nfs-backup storage class have been removed.

**Pull request merged**